### PR TITLE
feat: make hasMath false by default

### PR DESCRIPTION
**Pull request by gabrielfs7**
https://oat-sa.atlassian.net/browse/AUT-2082

### Solution

hasMath is false by default now and only changed on the scope of taoDeliverConnect installation

Documentation: https://github.com/oat-sa/tao-core/blob/feat/AUT-2082/feature-flag-visibility-tao-advance/models/classes/featureFlag/README.md

### TODO 

- [x] Add documentation
- [x] Validate solution with FE
- [x] Validate that installation works

## Related PRs

- https://github.com/oat-sa/extension-tao-deliver-connect/pull/116
- https://github.com/oat-sa/tao-core/pull/3530

### CODE REVIEW

File: migrations/Version202205181148531101_taoQtiItem.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                sprintf(
+                    "Applied the following configuration to `%s`\n%s",
+                    ExtendedTextInteractionConfigurationRegistry::class,
+                    json_encode($setValues)
+                )
+            )
+                : Report::createError(
+                sprintf('No values set to `%s`', ExtendedTextInteractionConfigurationRegistry::class)
+            )
+        );
```
Comment (hectoras) [inline migrations/Version202205181148531101_taoQtiItem.php:58]: suggestion: if-else may be more readable (same as https://github.com/oat-sa/extension-tao-itemqti/pull/2132/files#r875942661)

```suggestion
        if ($setValues) {
            $this->addReport(
                Report::createSuccess(
                    sprintf(
                        "Applied the following configuration to `%s`\n%s",
                        ExtendedTextInteractionConfigurationRegistry::class,
                        json_encode($setValues)
                    )
                )
            );
            
            return;
        }
        
        $this->addReport(
            Report::createError(
                sprintf('No values set to `%s`', ExtendedTextInteractionConfigurationRegistry::class)
            )
        );
```

**Review by hectoras — APPROVED** [review]

**Review by hectoras — COMMENTED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not master
- [x] Kitchen links are not exposed in the description
- [x] Commits are following conventional commits

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: migrations/Version202205181148531101_taoQtiItem.php (hunk 1 above)
Comment (hectoras) [reply migrations/Version202205181148531101_taoQtiItem.php:58]: (For consistency with the previous one, maybe it is better to have the error condition first :D)


```suggestion
        if (!$setValues) {
            $this->addReport(
                Report::createError(
                    sprintf('No values set to `%s`', ExtendedTextInteractionConfigurationRegistry::class)
                )
            );
            return;
        }

        $this->addReport(
            Report::createSuccess(
                sprintf(
                    "Applied the following configuration to `%s`\n%s",
                    ExtendedTextInteractionConfigurationRegistry::class,
                    json_encode($setValues)
                )
            )
        );
```

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**